src/Shared/Microsoft.DotNet.Scaffolding.Shared/Project/ProjectModifierHelper.cs

```diff
@@ -41,43 +41,31 @@ internal static async Task<bool> IsMinimalApp(IModelTypesLocator modelTypesLocat
         // Returns true when there is no Startup.cs or equivalent
         internal static async Task<bool> IsMinimalApp(CodeAnalysis.Project project)
         {
-            if (project != null)
+            var startupDocument = project?.Documents.Where(d => d.Name.EndsWith("Startup.cs")).FirstOrDefault() ?? null;
+            if (startupDocument != null)
```

Comment:
If we find a startupDocument aka Startup.cs exists, this if condition is successful, and we return true, but this application would not be a minimal app. The bool should be reversed.

Reply:
Oh, true, good catch